src/views/Dashboard/GeneticAnalyst/Services/index.vue

```diff
@@ -239,7 +262,18 @@ export default {
       await this.getDeleteServiceFee()
     },
 
-    async onDelete() {
+    onDelete() {
+      const activeOrder = this.orders?.data?.filter(order => order._source.status === "Paid")
+      const isActiveOrder = activeOrder?.some(order => order._source.service_id === this.serviceId)
```

Comment:
why do you need a filter method?

Reply:
filter to find active orders then look for the same serviceId, wdyt @bukandicki ?

Reply:
there is an update in this function without using the filter method